sources/packages/backend/apps/db-migrations/src/sql/Announcements/Create-announcements-table.sql

```diff
@@ -0,0 +1,34 @@
+CREATE TABLE IF NOT EXISTS sims.announcements (
+    id SERIAL PRIMARY KEY,
+    message_title VARCHAR(100) NOT NULL,
+    message VARCHAR(200) NOT NULL,
+    start_date TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW (),
+    end_date TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW (),
```

Comment:
There is no requirement to have a default value here.

Reply:
updated

Reply:
Start and end date are still having default values. 
![image](https://github.com/user-attachments/assets/6e48bf80-186d-4ef4-9073-22ce960a0d19)

Reply:
moved the defaults over to the created and updated as I'd accidentally removed the wrong ones the first time